### PR TITLE
ARTEMIS-1390 Start HawtIO in offline mode

### PR DESCRIPTION
**Pull request by mtaylor**

### CODE REVIEW

File: artemis-distribution/src/main/resources/bin/artemis [hunk 1]
```diff
@@ -94,5 +94,6 @@ exec "$JAVACMD" $JAVA_ARGS $ARTEMIS_CLUSTER_PROPS \
     -classpath "$CLASSPATH" \
     -Dartemis.home="$ARTEMIS_HOME" \
     -Djava.library.path="$ARTEMIS_HOME/bin/lib/linux-$(uname -m)" \
+    -Dhawtio.offline="true" \
```
Comment (michaelandrepearce) [inline artemis-distribution/src/main/resources/bin/artemis]: Is this needed as it's set in jvm args in profile

File: artemis-distribution/src/main/resources/bin/artemis.cmd [hunk 2]
```diff
@@ -48,7 +48,7 @@ echo.
 :RUN_JAVA
 
 rem "Set Defaults."
-set JAVA_ARGS=-XX:+UseParallelGC -XX:+AggressiveOpts -XX:+UseFastAccessorMethods -Xms512M -Xmx1024M
+set JAVA_ARGS=-XX:+UseParallelGC -XX:+AggressiveOpts -XX:+UseFastAccessorMethods -Xms512M -Xmx1024M -Dhawtio.offline="true"
```
Comment (michaelandrepearce) [inline artemis-distribution/src/main/resources/bin/artemis.cmd]: Is this needed as it's set in jvm args in profile

File: artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis.cmd [hunk 3]
```diff
@@ -64,6 +64,7 @@ set JVM_ARGS=%JVM_ARGS% -Dartemis.instance=%ARTEMIS_INSTANCE%
 set JVM_ARGS=%JVM_ARGS% -Ddata.dir=%ARTEMIS_DATA_DIR%
 set JVM_ARGS=%JVM_ARGS% -Djava.util.logging.manager=%ARTEMIS_LOG_MANAGER%
 set JVM_ARGS=%JVM_ARGS% -Dlogging.configuration=%ARTEMIS_LOGGING_CONF%
+set JVM_ARGS=%JVM_ARGS% -Dhawtio.offline="true"
```
Comment (michaelandrepearce) [inline artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis.cmd]: Is this needed as it's set in jvm args in profile

File: artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis [hunk 4]
```diff
@@ -119,5 +119,6 @@ exec "$JAVACMD" \
     -Ddata.dir="$ARTEMIS_DATA_DIR" \
     -Djava.util.logging.manager="$ARTEMIS_LOG_MANAGER" \
     -Dlogging.configuration="$ARTEMIS_LOGGING_CONF" \
+    -Dhawtio.offline="true" \
```
Comment (michaelandrepearce) [inline artemis-cli/src/main/resources/org/apache/activemq/artemis/cli/commands/bin/artemis]: Is this needed as it's set in jvm args in profile with all the other hawtio flags

**Comment by mtaylor** [comment]
@michaelandrepearce Yes these are not needed.  I've updated accordingly, cheers.

**Comment by michaelandrepearce** [comment]
+1 LGTM. @mtaylor thanks for this :)

**Pull request closed without merging**